src/sentry/uptime/rdap/query.py

```diff
@@ -110,11 +121,14 @@ def resolve_rdap_bootstrap_registry() -> ServiceArray:
     return services
 
 
-def resolve_hostname(hostname: str) -> str:
+def resolve_hostname(hostname: str) -> str | None:
     """
-    Resolves a hostname to a ipv4 address.
+    Resolves a hostname to a ipv4 address. Returns none if the address cannot be resolved.
     """
-    return gethostbyname(hostname)
+    try:
+        return gethostbyname(hostname)
+    except OSError:
```

Comment:
Does this actually raise an `OSError`? The sentry error has `gaierror`

Reply:
the base is `OSError`. I think it raises a few other errors actually